### PR TITLE
lock the scroll on body when opening drawer

### PR DESCRIPTION
**Pull request by wakooka**

### CODE REVIEW

**Comment by wakooka** [comment]
> Don't forget the changelog
We need a bot for that

File: src/drawer/index.tsx [hunk 1]
```diff
@@ -102,9 +103,15 @@ export default class Drawer extends PureComponent <DrawerProps, DrawerState> {
...
 
-  scrollReset = () => {
+  scrollLock = () => {
     if (canUseDOM) {
-      window.scroll(0, 0)
+      document.documentElement.classList.add('scroll-lock')
+    }
+  }
+
+  scrollRestore = () => {
```
Comment (pith) [inline src/drawer/index.tsx]: unlock ?

**Comment by pith** [comment]
Looks like a coding night project ;)

File: src/drawer/index.tsx (hunk 1 above)
Comment (wakooka) [reply src/drawer/index.tsx]: done

**Comment by Bidjii** [comment]
We could also create a PR template with checboxes

**Pull request merged**